packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -1070,8 +1070,6 @@ function clientMetadataStore(
 
 /** Provides LRU functionality for IndexedDB persistence. */
 export class IndexedDbLruDelegate implements ReferenceDelegate, LruDelegate {
-  private inMemoryPins: ReferenceSet | null = null;
```

Comment:
I removed the `inMemoryPins` redirection to the LocalStore. This was possible because LocalStore now directly pins all documents in a View via `addReference`.